src/git/config.go

```diff
@@ -276,7 +277,7 @@ func (c *Configuration) IsMainBranch(branchName string) bool {
 
 // IsOffline indicates whether Git Town is currently in offline mode
 func (c *Configuration) IsOffline() bool {
-	config := c.getGlobalConfigValue("git-town.offline")
+	config := c.getLocalOrGlobalConfigValue("git-town.offline")
```

Comment:
Should this be local? I thought we always used global

Reply:
For tests I set the local repo to offline. Otherwise the tests interfere with each other.
I can also see a business case for taking individual repos offline repos. For example when not on VPN and not having access to the company-internal Git server, but still to public Github.

Reply:
Currently we only have the UI for setting it at the global level. If we are going to start reading it from a local config, we should also update the config. And appears the ruby tests use global config. If we update the go tests to run as a new test user each time they should not interfere with each other (as global configs are per user)

Reply:
Forgot how we did this in the ruby feature tests and it was by setting the HOME env variable (as that is where the global git config is placed)

Reply:
Thanks for helping investigate this! I am now setting HOME to the local directory so that Git uses individual global config per test, as we had done in the Ruby tests. This solves a number of other issues where tests changed the global Git configuration.